src/index.ts

```diff
@@ -75,6 +80,12 @@ class Plugin implements TransformNode {
    */
   cachePath?: string;
 
+  /**
+   * `input` and `output` are Proxy objects to the `fs` operations.
+   */
+  input?: object;
```

Comment:
Do we plan to make better types for these? Seems like that might be nice (in order to catch when folks extending with typescript are using methods that aren't supported)

Reply:
We should find a way to add a type definition to the `FSMerger` and `broccoli-output-wrapper` and then use it here.